code/game/objects/items/weapons/weaponry.dm

```diff
@@ -78,22 +77,32 @@ var/highlander_claymores = 0
 /obj/item/weapon/claymore/highlander/New()
 	..()
 	START_PROCESSING(SSobj, src)
-	highlander_claymores++
 
 /obj/item/weapon/claymore/highlander/Destroy()
-	STOP_PROCESSING(SSobj, src)
-	highlander_claymores--
 	if(nuke_disk)
 		nuke_disk.forceMove(get_turf(src))
 		nuke_disk.visible_message("<span class='warning'>The nuke disk is vulnerable!</span>")
 		nuke_disk = null
+	STOP_PROCESSING(SSobj, src)
 	return ..()
 
+/obj/item/weapon/claymore/highlander/process()
+	if(ishuman(loc))
+		var/mob/living/carbon/human/H = loc
+		loc.layer = 5 //NO HIDING BEHIND PLANTS FOR YOU, DICKWEED
+		H.bleedsuppress = 1 //AND WE WON'T BLEED OUT LIKE COWARDS

```

Comment:
Hell if I know. Different maintainers like different things and it's used interchangeably throughout the entire code.